kythe/proto/explore.proto

```diff
@@ -167,36 +165,28 @@ message TypeHierarchyReply {
 
 // Requests the functions that call the specified function.
 message CallersRequest {
-  string function_ticket = 1;
-
-  NodeFilter node_filter = 2;
+  repeated string function_ticket = 1;
```

Comment:
Do we want the word "function" here, given that there are callable things whose kind is not `function`?

Reply:
I don't mind leaving out "function" here on the presumption that we may expand the set of nodes in the callgraph to include non-functions, sure.  (At the moment the postprocessing pipeline only includes functions, though.)